spec/open-api.yml

```diff
@@ -1874,6 +1879,17 @@ paths:
       responses:
         "200":
           description: OK
+          headers:
+            Link:
+              schema:
+                type: string
+                example: </v1/projects/foo/agents?before=bar>; rel="next"
+              description: RFC5988 with `rel="next"`.
+          links:
+            NextAgentsPage:
+              operationId: agents
+              parameters:
+                before: "$response.header.Link#rel=next"
```

Comment:
The part after the hash `#` is not totally correct. I didn't found any tool that support it. But it is also not incorrect. I left it there for documentation puposes.

Reply:
Maybe we could remove the `links` field part. For example GitHub's API uses `Link` header bases pagination https://github.com/github/rest-api-description/blob/main/descriptions/api.github.com/api.github.com.yaml but it doesn't document the links field.